undetected_chromedriver/cdp.py

```diff
@@ -3,7 +3,7 @@
 
 import json
 import logging
-from collections import Mapping, Sequence
+from collections.abc import Mapping, Sequence
```

Comment:
```suggestion
import sys
import json
import logging
exec('from collections.abc import Mapping, Sequence') if sys.version_info >= (3, 3) else exec('from collections import Mapping, Sequence')
```
This should help with forward and backward compatibility! Felt a bit proud coming up with a pythonic ternary operator :)